assembly/runtime/api/others.ts

```diff
@@ -66,6 +66,6 @@ export function Metadata_metadata(data: i32, len: i32): u64 {
 export function System_account_nonce(data: i32, len: i32): u64 {
     const input = Serialiser.deserialiseInput(data, len);
     const who = BytesReader.decodeInto<AccountIdType>(input);
-    const nonce = System.accountNonce(who);
+    const nonce = SystemStorageEntries.AccountNonce().get(who);
```

Comment:
Is there a way that we could do something in the form of:
`System.StorageEntries.AccountNonce().get()` or at least have a way to reference that we are accessing the storage entries in the system module.

Reply:
I guess that would be using `module.exports`?